cmd/mattermost/commands/user.go

```diff
@@ -398,6 +408,126 @@ func usersToBots(args []string, a *app.App) {
 	}
 }
 
+func getUpdatedPassword(command *cobra.Command, a *app.App, user *model.User) (string, error) {
+	password, err := command.Flags().GetString("password")
+	if err != nil {
+		return "", fmt.Errorf("Unable to read password. Error: %s", err.Error())
+	}
+
+	if password == "" {
+		return "", errors.New("Password is required.")
+	}
+
+	return password, nil
+}
+
+func getUpdatedUserModel(command *cobra.Command, a *app.App, user *model.User) (*model.User, error) {
+	username, err := command.Flags().GetString("username")
+	if err != nil || username == "" {
```

Comment:
nit: There is no point on checking the error here, as we are not returning it and `username` will be empty (the other condition) if there is an error. This can be simplified to:

```go

username, _ := command.Flags().GetString("username")
if username == "" { 
    ...
}
```